Source/ACE.Entity/Enum/EnvironChangeType.cs

```diff
@@ -0,0 +1,48 @@
+
+namespace ACE.Entity.Enum
+{
+    public enum EnvironChangeType
+    {
+        Clear               = 0x00,
```

Comment:
enum duplicated from existing ACE.Server.Network.Enum.EnvironChangeType

Reply:
deleted the one from network